src/main/java/technology/dice/dicefairlink/discovery/members/awsapi/AwsApiReplicasFinder.java

```diff
@@ -58,23 +67,48 @@ private DBCluster describeCluster(String clusterId) {
   }
 
   private Set<String> replicaMembersOf(DBCluster cluster) {
-    final Optional<DBClusterMember> writer =
-        cluster.dbClusterMembers().stream().filter(member -> member.isClusterWriter()).findAny();
-    final Set<String> replicaIds =
-        cluster.dbClusterMembers().stream()
-            .filter(dbClusterMember -> isDbWriter(writer, dbClusterMember))
-            .map(dbClusterMember -> dbClusterMember.dbInstanceIdentifier())
-            .collect(Collectors.toSet());
+    try {
+      DescribeDbInstancesRequest request =
+          DescribeDbInstancesRequest.builder()
+              .filters(
+                  Filter.builder()
+                      .name(DB_CLUSTER_ID_FILTER)
+                      .values(cluster.dbClusterIdentifier())
+                      .build())
+              .build();
+      final Optional<DBClusterMember> writer =
+          cluster.dbClusterMembers().stream().filter(member -> member.isClusterWriter()).findAny();
```

Comment:
use `Stream.anyMatch(...)`

Reply:
That will only give me a boolean, I need the actual instance id